packages/code-editor/package.json

```diff
@@ -48,7 +52,7 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-is": "^16.13.1",
-    "styled-components": "^5.2.1",
```

Comment:
might be worth bumping @types/styled-components as well

Reply:
We're up to date with @looker/components. I tried bumping the @types but it threw errors on how we extend Looker components